feature-extensions/multiaccount/setup.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python2
```

Comment:
Please convert all Python code to Python3. The next release should have a Python3 installer #382  and we are dropping Python2, as it is being deprecated and will no longer be supported as of Jan 1, 2020.

Reply:
Done